### PR TITLE
Add missing states and cities to PAN, GTM, and CRI

### PR DESCRIPTION
**Pull request by camilavcoutinho**
Added missing Guatemala ('GTM'), Costa Rica ('CRI') and Panama ('PAN') states and cities. Tracked by tasks [LOC-8933](https://vtex-dev.atlassian.net/browse/LOC-8933), [LOC-8934](https://vtex-dev.atlassian.net/browse/LOC-8934), and [LOC-8935](https://vtex-dev.atlassian.net/browse/LOC-8935).

### CODE REVIEW

File: react/country/PAN.js [hunk 1]
```diff
@@ -0,0 +1,1083 @@
+import { THREE_LEVELS } from '../constants'
+import { thirdLevelPostalCodes } from '../transforms/postalCodes'
+import {
+  getOneLevel,
+  getTwoLevels,
+  getThreeLevels,
+} from '../transforms/addressFieldsOptions'
+
+const countryData = {
```
Comment (lucasecdb) [inline react/country/PAN.js:9]: can you move this country data to the `data` folder? so we can keep the country files a bit more lean

Comment (sheilagomes) [reply react/country/PAN.js]: Done!

**Review by lucasecdb — APPROVED** [review]

**Review by sheilagomes — APPROVED** [review]

**Review by kaio-donadelli — APPROVED** [review]
LGTM

**Review by kaio-donadelli — APPROVED** [review]
LGTM

**Review by lucasecdb — APPROVED** [review]

**Pull request merged**